airdrop/application/services/user_notification_service.py

```diff
@@ -18,7 +18,8 @@ def subscribe_to_notifications(self, inputs):
             validate(instance=inputs, schema=schema,
```

Comment:
@anandrgitnirman on JSON request input add `airdrop_id` as a required parameter

Reply:
ddint want to make it mandatory in which case notifications will be sent out for all airdrop events